### DISCUSSION TITLE
Help generating public key from seed using eth_secp256k1

### DISCUSSION THREAD

**Discussion by 0xForerunner** [RustCrypto/elliptic-curves#923] [Q&A]
I'm a touch lost at the correct way to go about this. I know K256 is a rust implementation of secp256k1, but is there anything in this repo that can specifically help me with eth_secp256k1? Sorry for the vague question, If anyone could point me in the right direction that would be very helpful.

**Comment by tarcieri** [comment]
You'll probably want something like ethers-rs as an Ethereum library built on top of `k256`: https://github.com/gakonst/ethers-rs

**Reply by 0xForerunner** [reply]
Awesome thanks for the direction! I'll have a look through there.
